resources/Components/LazyLoadImage/index.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState, useCallback } from 'react'
+
+export const LazyLoadImage = ({ src, placeholder, loadError, ...props }) => {
+  const [imgSrc, initImg] = useState(placeholder || src)
+  const onLoad = useCallback(() => {
+    initImg(src)
+  }, [src])
+  const onError = useCallback(() => {
+    initImg(loadError || placeholder)
+  }, [loadError, placeholder])
+  useEffect(() => {
+    const imageObj = new Image()
```

Comment:
does this create another image element and another http call to the server to fetch the image?